src/main/java/com/epam/ta/reportportal/ws/model/dashboard/UpdateDashboardRQ.java

```diff
@@ -35,8 +36,9 @@
 @JsonInclude(Include.NON_NULL)
 public class UpdateDashboardRQ extends SharableEntityRQ {
 
-	@JsonProperty(value = "name")
-	@Size(min = ValidationConstraints.MIN_NAME_LENGTH, max = ValidationConstraints.MAX_DASHBOARD_NAME_LENGTH)
+	@NotBlankWithSize(min = ValidationConstraints.MIN_NAME_LENGTH, max = ValidationConstraints.MAX_DASHBOARD_NAME_LENGTH)
+	@JsonProperty(value = "name", required = true)
+	@ApiModelProperty(required = true)
```

Comment:
Should it be always required?

Reply:
As i understand, all params should be passed in case of both create and update. Can be discussed with Volha.

Reply:
It's better to discuss it with UI team and ask either they always send it or not.

Reply:
i've checked on dev env: now all the fields are sent on update, can discuss with UI team as well if it's needed